models/workday__employee_overview.sql

```diff
@@ -31,8 +32,8 @@ worker_employee_enhanced as (
         case when days_of_employment >= 365*20 and is_user_active then true else false end as is_current_employee_twenty_years,
         case when days_of_employment >= 365*30 and is_user_active then true else false end as is_current_employee_thirty_years
     from int_worker_base
-    left join int_worker_personal_details using(worker_id)
-    left join int_worker_position_enriched using(worker_id)
+    left join int_worker_personal_details using(worker_id) 
+    left join int_worker_position_enriched using(worker_id) 
```

Comment:
Same comment about using a left join as opposed to using.

Reply:
Updated.